db/seeds.rb

```diff
@@ -7,6 +7,7 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+require 'json'
```

Comment:
Did you need this? Rails should auto-load this library.

Reply:
This is done